src/it/resources-noncompilable/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputFormattedWhitespaceAroundWhen.java

```diff
@@ -50,40 +26,40 @@ void test2(Object o) {
 
     switch (o) {
       case Integer i when (i == 0) -> {}
-      // 2 violations above:
-      //  ''{' is not followed by whitespace.'
-      //  ''}' is not preceded with whitespace.'
```

Comment:
Hmmm this bug was known but it didn't had an issue opened for it. Glad that we were able to solve it now.

Reply:
@Zopsss , testing base that you did at summer is awesome to prevent any problems.
We missed it as a lot of problems and we eliminated vast majority of them. But some still missed, it is ok. Main point that we unlikely to leak new regression.